lib/livebook_web/live/session_live/bin_component.ex

```diff
@@ -36,21 +36,16 @@ defmodule LivebookWeb.SessionLive.BinComponent do
         Bin
       </h3>
       <div class="w-full flex-col space-y-5">
-        <p class="text-gray-700">
-          Here you can find all the deleted cells from this notebook session.
-        </p>
-        <%= if @bin_entries == [] do %>
-          <div class="p-5 flex space-x-4 items-center border border-gray-200 rounded-lg">
-            <div>
-              <.remix_icon icon="windy-line" class="text-gray-400 text-xl" />
-            </div>
-            <div class="text-gray-600">
-              There are currently no cells in the bin.
-            </div>
-          </div>
-        <% else %>
-          <form phx-change="search" onsubmit="return false" phx-target={@myself}>
-            <input class="input"
+        <div class="flex justify-between items-center">
+          <p class="text-gray-700 basis-4/5">
+            Find all your deleted cells from this notebook session
+          </p>
+          <form phx-change="search" onsubmit="return false" phx-target={@myself}
+            class={"mt-1 relative #{if @bin_entries == [], do: "hidden"}"}>
```

Comment:
If there are no entries it doesn't make sense to render the search form, right? So instead of hiding it with CSS, we can render conditionally and never send it, or is there a reason I miss? :)

Reply:
Nop! My only observation is that the original design showed the search even without cells. But I don't think it makes sense to show it and it looks so much cleaner without it.